worlds/sm64ex/Options.py

```diff
@@ -127,6 +127,33 @@ class MoveRandomizerActions(OptionSet):
     valid_keys = [action for action in action_item_table if action != 'Double Jump']
     default = valid_keys
 
+sm64_options_groups = [
+    OptionGroup("Logic Options", [
+        AreaRandomizer,
+        BuddyChecks,
+        ExclamationBoxes,
+        ProgressiveKeys,
+        EnableCoinStars,
+        EnableMoveRandomizer,
+        MoveRandomizerActions,
+        StrictCapRequirements,
+        StrictCannonRequirements,
+        StrictMoveRequirements,
+    ]),
+    OptionGroup("Star Options", [
+        AmountOfStars,
+        FirstBowserStarDoorCost,
+        BasementStarDoorCost,
+        SecondFloorStarDoorCost,
+        MIPS1Cost,
+        MIPS2Cost,
+        StarsToFinish,
+    ]),
+OptionGroup("Goal Options", [
+        CompletionType,
+    ]),
```

Comment:
```suggestion
```
Having Completion Type be this low and alone is a funky decision to make. I would probably recommend leaving it out of the option groups and letting it go to the "Game Options" group since it is more fundamental to the randomizer.